<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUsersWorkersInventory</class>
 <widget class="QMainWindow" name="MainUsersWorkersInventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инвентаризация</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/img/logopm.png</normaloff>:/images/img/logopm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
border-image: url(:/images/img/fonpril.png);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>131</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbAdd">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить документ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPrint">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>550</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Печать позиции</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Пользователь:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblUser">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Роль:                </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPost">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tbSklad">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>691</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить документ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPrintFilter">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>580</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Печать фильтра</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
